packages/core/src/Stores/client-store.js

```diff
@@ -1989,7 +1989,7 @@ export default class ClientStore extends BaseStore {
             const redirect_url = sessionStorage.getItem('redirect_url');
             if (
                 is_pre_appstore === 'true' &&
-                redirect_url?.endsWith('/') &&
+                (redirect_url?.endsWith('/') || redirect_url?.endsWith('/bot')) &&
```

Comment:
I'm not quite sure if this check `redirect_url?.endsWith('/')` is necessary, but I just left it here and added to it, incase removing it could break something